lib/client/conntest/database.go

```diff
@@ -184,7 +184,9 @@ func (s *DatabaseConnectionTester) TestConnection(ctx context.Context, req TestC
 		return nil, trace.Wrap(err)
 	}
 
-	if pingErr := databasePinger.Ping(ctx, ping); pingErr != nil {
+	pingCtx, cancel := context.WithTimeout(ctx, 10*time.Second)
```

Comment:
Can we use this const instead?
It's a bit higher (30s) but used in a lot of places.
https://github.com/gravitational/teleport/blob/bfaa3340e8a71a326baafaa645dd231ec0857395/api/defaults/defaults.go#L32